pkg/pop/poptokenutils.go

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package pop
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/confidential"
+	"github.com/AzureAD/microsoft-authentication-library-for-go/apps/public"
+)
+
+func AcquirePoPTokenInteractive(
+	popClaims map[string]string,
+	scopes []string,
+	authority,
+	clientId string,
+) (string, int64, error) {
+	client, err := public.New(clientId, public.WithAuthority(authority))
+	if err != nil {
+		return "", -1, fmt.Errorf("unable to create public client: %w", err)
+	}
+	result, err := client.AcquireTokenInteractive(
+		context.Background(),
```

Comment:
It is usually a good practice to get this context from the caller. This can be a parameter of the function

Reply:
Updated to pass context from caller